TestFoundation/TestNSRange.swift

```diff
@@ -126,4 +128,52 @@ class TestNSRange : XCTestCase {
             XCTAssertEqual(NSStringFromRange(range), string)
         }
     }
+    
+    private func _assert<S: StringProtocol, R: RangeExpression>(
```

Comment:
Can we give this a more specific name to indicate that it is specialized for the below test?